tests-builds-from-more-prs/README.md

```diff
@@ -0,0 +1,76 @@
+- we would need to have a dynamic way of specifying these `runtime_dependencies`
+- since we want concrete packages to be installed in TF (we pass the build ID and NVRs), this wouldn't be enough
+  - if users build all the PRs in one Copr repo (e.g.
+    [leapp](https://github.com/oamg/leapp/blob/351cfe75ce1a7e10d7e565d9fa0bdc946a9af66d/.packit.yaml#L9) and
+    [leapp-repository](https://github.com/oamg/leapp-repository/blob/8b228941d34b3fcf4322353806fdcafa9fbba550/.packit.yaml#L26) build their PRs in one common Copr repo)
+    we cannot enforce with this solution what should be installed during tests (build ID, NVRs).
+
+## Possible implementation
+
+### Requirements
+
+The requirement would be to have enabled Packit Copr build job on both repositories where the PRs are happening.
+Then, the testing farm test job could be triggered by comment specifying the other pull request ID
+and we would rely on builds being already done by Packit.
+
+### Implementation
+
+We should be able to get Copr build ID and package NVRs via our database having the other PR ID:
+`PullRequestModel.get_or_create(pr_id=xy, namespace=xy, repo_name=xy, project_url=xy).get_copr_builds()`
+and determine the latest build comparing one of the stored times (task_accepted, build_submitted).
+
+With having the Copr build ID and package NVRs of the other PR Copr build, we could just pass these
+into the Testing farm payload.
+
+Process:
+
+1. Try to get the PR specified in the comment from the DB - if not able to find, report this to user
+2. Get the latest Copr build for the PR from the DB - if no Copr build, report this to user
+3. Append artifact including the ID of the Copr build and NVRs to the payload for Testing Farm
+4. Trigger the tests
+5. When reporting the status, make sure it is done on the "main" PR (probably corresponding to the
+   first one build in the artifacts)
+
+### Pipeline model
+
+Currently, each pipeline model can be linked to one Copr build model and one TF model at most.
+
+- Could we not store the information about the other PR build? Consequences => the other PR build would not
+  be displayed in the dashboard, we would lose this information.
+- Or would making changes in the DB schema make sense taking into consideration the main testing farm job use case?
+  (e.g. new additional_builds field or making it possible to store more builds)
+
+### Retriggering
+
+How can a testing farm job be retriggered:
+
+1. via comment - we would require each time to explicitly pass the PR related attributes
+2. via Github check - we would trigger the tests only with the build from the current PR
+
+### Syntax of the command
+
+questions:
+
+1. do we want to support specifying also Git forge?
+2. do we want to do this only for PRs? (or also Copr builds from commits/releases from other repo)
+
+suggestions:
+
+1. `packit test --with-pr --pr-id 123 --repository repo --namespace namespace (--forge github.com)`
+
+2. `packit test --with-pr namespace/repo#123`
```

Comment:
+1 to start here, use the native GitHub syntax and extend this in future to other gitforges (or more options)